buildSrc/src/test/java/com/soundcloud/reflect/DaggerReflectPluginIntegrationTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.soundcloud.reflect
+
+import com.google.common.truth.Truth.assertThat
+import org.gradle.testkit.runner.GradleRunner
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import java.io.File
+
+class DaggerReflectPluginIntegrationTest {
+    @get:Rule
+    val testProjectDir = TemporaryFolder()
+
+    @Test
+    fun testPluginWithoutDaggerReflect() {
+        val fixtureName = "java-module"
+        writePluginBuildGradle()
+
+        writeSettingsGradle(fixtureName)
+
+        copyProjectFixture(fixtureName)
```

Comment:
how about extracting those lines? seems you use them more than ones. 

keep the tests clean
![](https://developeronfire.com/assets/images/UncleBobMartin.jpg)